drivers/scheduler/k8s/k8s.go

```diff
@@ -76,12 +76,14 @@ var (
 	namespaceRegex = regexp.MustCompile("{{NAMESPACE}}")
 )
 
-type k8s struct {
-	specFactory    *spec.Factory
-	nodeDriverName string
+//K8s  The kubernetes structure
+type K8s struct {
```

Comment:
We can make it as `K8S` instead of `K8s` everywhere.

Reply:
At present , I have not  incorporated this change. 
If others, feel it is good  to have it changed to upper case, will incorporate it
@harsh-px, let me know, what is your opinion on this